cartographer/cloud/proto/map_builder_service.proto

```diff
@@ -70,6 +71,7 @@ message SensorData {
 
 message AddTrajectoryResponse {
   int32 trajectory_id = 1;
+  string client_id = 2;
```

Comment:
seems unused? (actually, the entire response seems unused)

Reply:
done